### DISCUSSION TITLE
Wildcard

### DISCUSSION THREAD

**Discussion by dicodaniel** [StevenBlack/hosts#2262] [Ideas]
I see in the gambling list de.betano.com, Is it possible to add a wildcard to the domain as the domain still loads for different countries (ro.betano.com by example) or could you just add the domain I used as an example?

**Comment by StevenBlack** [comment]
Thank you for this question @dicodaniel.

Hosts files are as old as the internet itself. Hosts files are a dictionary map of domain names to IP addresses. No more, no less.

Hosts files do not support wildcards. Each domain, subdomain, sub-subdomain, is distinct.  And this is probably never going to change 😄

**Comment by StevenBlack** [comment]
@dicodaniel that said, [OpenDNS](https://www.opendns.com/) offers wildcard-based blocking, of a sort.

See https://support.opendns.com/hc/en-us/articles/227986747-How-do-I-add-Domain-Wildcards-to-My-Block-or-Allow-Lists-
